src/Runner.Worker/ExecutionContext.cs

```diff
@@ -576,12 +582,6 @@ public void InitializeJob(Pipelines.AgentJobRequestMessage message, Cancellation
             // File table
             FileTable = new List<String>(message.FileTable ?? new string[0]);
 
-            // Expression functions
```

Comment:
seems fine to remove this?

Reply:
I think it should be fine, we need to make sure either the FF is on for all rings first before we push this runner to all rings. :)